### ISSUE TITLE
slack: feature request: viewing multiple workspaces

### ISSUE THREAD

**Issue by aflat** [eul-im/eul#105]
Instead of having a big list of slack workspaces, with workspace names next to the channel, it might be nice to have a divider line separating each workspace

**Comment by voltapp2** [comment]
Yes, this is a very frequently requested feature. It will be implemented this week ;)
